mono/mini/aot-compiler.c

```diff
@@ -10649,7 +11019,16 @@ mono_compile_assembly (MonoAssembly *ass, guint32 opts, const char *aot_options)
 		}
 	}
 
-	load_profile_files (acfg);
+	if (acfg->aot_opts.profile_file)
+		load_profile_file (acfg, acfg->aot_opts.profile_file);
+
+	{
+		int method_index;
```

Comment:
This is just a first version made to speed us roslyn. We can add an option to exclude non-profile methods. In roslyn's case, that would mean that any csc method not hit during the profile run will have to be JITted.